src/runtime/data/kata-collect-data.sh.in

```
@@ -523,7 +527,7 @@ show_containerd_shimv2()
 {
 	start_section "Containerd shim v2"
 
-	local cmd="${containerd_shim_v2_name} --version"
+	local cmd="${containerd_shim_v2} --version"
```

Hmm, does the shim require the `$SCRIPT_DIR` path? Then `containerd_shim_v2=$(…)` should be adapted too. Or doesn't it? Then I'm not sure why this change is necessary.

---

This is not needed for this bug fix.
But while I was creating the commit to run with the full path, I was also fixing this one.

However, looking at the results of the run again, kata-collect-data.sh seems to silently fail `containerd-shim-kata-v2 --version`. Therefore,  I thought it would be a good idea to fix containerd-shim-kata-v2 as well.